packages/docs/docs/billing/insurance-eligibility-checks.md

```diff
@@ -0,0 +1,135 @@
+import ExampleCode from '!!raw-loader!@site/..//examples/src/billing/insurance-eligibility-checks.ts';
+import MedplumCodeBlock from '@site/src/components/MedplumCodeBlock';
+
+# Insurance Eligibility Checks
+
+Insurance eligibility checks determine whether a patient's insurance is active, in-network, and has applicable benefits. They ensure that the provider will ultimately get compensated by the patient's insurer for a specific product or service.
+
+## Use Cases
+
+Insurance eligibility checks cover a variety of use cases, but three of the most important are:
+
+1. Requests for prior authorization
+2. Confirmation of benefits
+3. Coordination of care
+
+### Requests for prior authorization
+
+Requests for prior authorization ensure that the patient is both covered and authorized to receive the requested care. This is used in cases where the insurer must ensure that the care is necessary before it is authorized. Often a patient must have a certain diagnosis or have attempted other care options prior to receiving authorization.
+
+### Confirmation of benefits
+
+Insurance eligibility checks used for a confirmation of benefits are similar to requests for prior authorization, but they are simply checking whether the benefits cover the requested care. This is often for elective care and the check is to ensure that the care will be covered but does not require authorization from the insurer.
```

Comment:
Do you mean service types? https://x12.org/codes/service-type-codes - check whether this policy requires a given service type?